lib/ansible/modules/cloud/digital_ocean/digital_ocean_space.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Aaron Smith <ajsmith10381@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: digital_ocean_space
+short_description: Manage spaces in DigitalOcean.
+description:
+    - Create and remove spaces in DigitalOcean.
+author: "Aaron Smith (@slapula)"
+version_added: "2.6"
+requirements: [ 'botocore', 'boto3' ]
+options:
+  name:
+    description:
+     - The name of the space.
+    required: true
+  state:
+    description:
+     - Whether the space should be present or absent.
+    default: present
+    choices: ['present', 'absent']
+  region:
+    description:
+    - Region where the space will reside.
+    default: 'nyc3'
+  canned_acl:
+    description:
+    - Canned ACL to apply to the bucket.
+    default: private
+    choices: ['private', 'public-read']
+  access_id:
+    description:
+    - Access ID used for authentication with Digital Ocean Spaces API (Default is DO_ACCESS_KEY_ID).
+    default: ''
+  secret_key:
+    description:
+    - Secret key used for authentication with Digital Ocean Spaces API (Default is DO_SECRET_ACCESS_KEY).
+    default: ''
+extends_documentation_fragment:
+    - digital_ocean.documentation
```

Comment:
I'm not sure if we should include **digital_ocean.documentation** since we don't use any documented parameters